src/System.Collections.Concurrent/pkg/System.Collections.Concurrent.pkgproj

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <ProjectReference Include="..\ref\System.Collections.Concurrent.csproj">
+    <ProjectReference Include="..\ref\System.Collections.Concurrent.builds">
       <SupportedFramework>net46;netcore50;netcoreapp1.0;$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
This should be updated to `<SupportedFramework>netcoreapp1.1;net463;$(AllXamarinFrameworks)</SupportedFramework>`